### ISSUE TITLE
[clfft] taking address of rvalue

### ISSUE THREAD

**Issue by acxz** [acxz/pkgbuilds#204]
```
/home/dale/.aur_build/trizen-dale/clfft/src/clFFT-2.12.2/src/library/generator.transpose.cpp: In function ‘clfftStatus clfft_transpose_generator::genSwapKernelGeneral(const FFTGeneratedTransposeNonSquareAction::Signature&, std::string&, std::string&, const size_t&, size_t)’:
/home/dale/.aur_build/trizen-dale/clfft/src/clFFT-2.12.2/src/library/generator.transpose.cpp:1092:95: error: taking address of rvalue [-fpermissive]
 1092 |         std::string smaller_dim_str = static_cast<std::ostringstream*>(&(std::ostringstream() << smaller_dim))->str();
      |                                                                         ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
/home/dale/.aur_build/trizen-dale/clfft/src/clFFT-2.12.2/src/library/generator.transpose.cpp:1093:93: error: taking address of rvalue [-fpermissive]
 1093 |         std::string dim_ratio_str = static_cast<std::ostringstream*>(&(std::ostringstream() << dim_ratio))->str();
      |                                                                       ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
make[2]: *** [library/CMakeFiles/clFFT.dir/build.make:160: library/CMakeFiles/clFFT.dir/generator.transpose.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:156: library/CMakeFiles/clFFT.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
:: Unable to build clfft - makepkg exited with code: 4
```

**Comment by acxz** [comment]
patched with [faf72ea](https://github.com/acxz/pkgbuilds/commit/faf72eaca4b63d7d2fbf548a92f5715b7941c08b)

**Comment by acxz** [comment]
closing issue as I'm not maintaining this package anymore.
